src/extension.ts

```diff
@@ -141,10 +145,6 @@ async function _activateExtension(
   await Promise.all([setupStorage(), setupContextValues()]);
   logger.info("Storage and context values initialized");
 
-  // determine the writeable tmpdir for the extension to use. Must be done prior
-  // to starting the sidecar, as it will use this tmpdir for sidecar logfile.
-  await WriteableTmpDir.getInstance().determine();
```

Comment:
Moved up before any logging during extension activation since extension logging needs this to be determined ahead of time